bootstrap/bootstrap.sh

```diff
@@ -4,10 +4,9 @@
 
 set -euo pipefail
 
-: "${bootstrap_path:?Missing bootstrap path}"
+here="$(dirname "$(readlink -f "$0")")"
+
 environment="${1}"
-export storageclass_path="${bootstrap_path}/storageclass"
-export namespaces_path="${bootstrap_path}/namespaces"
```

Comment:
Isn't this used in `bootstrap/namespaces/bootstrap.sh`?

Reply:
damn, nice catch! i've tried to make them a bit more self-contained but missed that one - fixed now

Reply:
Nice! yea, It's a bit odd that the parent script should provide the path.

Reply:
Fix pushed!